mifiel/document.py

```
@@ -108,17 +109,26 @@ def create_many_from_template(client, args={}):
   def save_file(self, path):
     url_ = self.url('{}/file').format(self.id)
     response = requests.get(url_, auth=self.client.auth)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)